components/shared/Team/index.tsx

```diff
@@ -8,7 +7,12 @@ import Item from './team-item';
 import styles from './styles.module.scss';
 import person from './item-interface';
 
-function Team({ infinite, items }) {
+interface Props {
+  items: [person];
```

Comment:
This isn't _quite_ the same thing as an array of person; it's actually an array with 1 item of the Person type. Use `Person[]` instead

Reply:
fixed